lib/customer.rb

```diff
@@ -0,0 +1,34 @@
+class Customer
+
+  attr_reader :basket
+
+  def initialize
+    @basket = []
+  end
+
+  def view_menu(takeaway)
+    takeaway.menu
+  end
+
+  def add_item(takeaway, _item)
+    fail 'Sorry that item is not on the menu' if takeaway.menu.key?(_item) == false
```

Comment:
You could use `fail 'Sorry that item is not on the menu' unless takeaway.menu.key?(_item)`